src/core/font_substitutions.js

```diff
@@ -238,13 +208,15 @@ const substitutionMap = new Map([
   [
     "ArialBlack-BoldItalic",
     {
-      alias: "ArialBlack-Italic",
+      alias: {
+        name: "ArialBlack-Italic",
+      },
```

Comment:
Shouldn't this be formatted as follows now?
```suggestion
      alias: "ArialBlack-Italic",
```